packages/stark-ui/src/modules/action-bar/components/action-bar-config.intf.ts

```diff
@@ -13,4 +13,10 @@ export interface StarkActionBarConfig {
 	 * If false, then action bar will not be present on the page (optional)
 	 */
 	isPresent?: boolean;
+
+	/**
+	 * Should the actions always be shown on the right side of the table (sticky)
+	 * Default: false
+	 */
+	isSticky?: boolean;
```

Comment:
I think we should not mix table stuff with the action bar.
There is no reason for a generic action bar to be sticky.
Could you please move this logic in the Input of the table component? 😊

Reply:
You are right, the reason I put it there is because `tableRowsActionBarConfig` is of this type and I didn't pay much attention to the name. 😅 I will add another (similar type) for the row actions, because it does make sense (to me) to add this property to `tableRowsActionBarConfig` or maybe rename it to `tableRowsActions`.

Reply:
Indeed, I agree to put this option in the `tableRowsActionBarConfig` 👍

Reply:
I updated the PR